installInstructions/install3rdPartyPackages.rst

```diff
@@ -72,17 +72,19 @@ OpenMPI
    On a cluster, the system administrator will have already compiled various versions of MPI on the system already.
    Do not build/install OpenMPI in this case, and simply load the correct MPI module.
 
-Download from the `OpenMPI <http://www.open-mpi.org/>`__ website and place the tarball in your packages directory (``$HOME/packages``).
-Then, unpack the source code:
+Download the desired version from the `OpenMPI <http://www.open-mpi.org/>`__ website and place the tarball in your packages directory, ``$HOME/packages``:
 
 .. prompt:: bash
 
-   cd $HOME/packages
-   wget <download URL>
+   wget https://download.open-mpi.org/release/open-mpi/<version>/openmpi-<version>.tar.gz
```

Comment:
This URL is slightly misleading because the first `<version>` is not the same as the second. Then again, I don't know if we have to be that specific about the URL.

Reply:
Yeah good point, in general I dislike putting direct URLs here because they may change without our notice. It's probably better to tell people to download the file (either `wget` in the terminal or within their browser) rather than hard-coding these.